README.md

```diff
@@ -48,3 +49,71 @@ docker-compose up -d matchbox
+- Invalid Transaction (422 Error): When a Health Event Request is sent via POST to `[base]/submitHealthEvent` that does not conform to the [DDCCSubmitHealthEvent profile](http://worldhealthorganization.github.io/ddcc/StructureDefinition/DDCCSubmitHealthEventRequest), the response should return 422 status.
+
+#### Generate Health Certificate
+
+This FHIR operation accepts a set of immunization data elements via a [FHIR Parameters resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html) and returns [DDCCDocument resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCDocument.html) as the response.
+To learn more about this transaction, refer to the [Generate Health Certificate OperationDefinition](https://worldhealthorganization.github.io/ddcc/OperationDefinition-DDCC-QuestionnaireResponse-generateHealthCertificate.html).
+
+The following assertions are made for this transaction:
+
+- Valid Transaction: When a valid Generate Health Certificate Request is sent via POST to `[base]/QuestionnaireResponse/$generateHealthCertificate`, the response should:
+  - Return 201 status.
+  - Return a valid [DDCCDocument resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCDocument.html) as the response body.
+- Invalid Transaction (422 Error): When a Generate Health Certificate Request is sent via POST to `[base]/QuestionnaireResponse/$generateHealthCertificate` that does not conform to the [DCCCGenerateHealthCertificateParameters profile](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html), the response should return 422 status.
+
+### DDCC:VS Registry Service
+
+#### Register Health Certificate
+
+This transaction is called by the Generate Health Certificate service when a new Submit Health Event is accepted. The request consists of a [DDCC Provide Document Bundle](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCProvideDocumentBundle.html), and the response is a transaction bundle. The transaction is based on the [MHD Provide Document Bundle transaction](https://profiles.ihe.net/ITI/MHD/ITI-65.html#2365412-message-semantics). To learn more about this transaction, refer to the [Register Health Certificate workflow in the WHO specification](https://worldhealthorganization.github.io/ddcc/transactions.html#register-health-certificate).
+
+The following assertions are made for this transaction:
+
+- Valid Transaction: When a valid Register Health Certicate Request is sent via POST to `[base]`, the response should:
+  - Return 200 status.
+  - Return a transaction bundle that has at least one entry for each entry in the request, in the same order, with an entry response location.
+- Invalid Transactions:
+  - 400 Error: When a POST request is sent to `[base]` that fails the basic FHIR validation rules, the response should return 400 status.
```

Comment:
A short note somewhere explaining what `[base]` refers to could be nice, especially since the service can technically be different depending on the transaction.